### ISSUE TITLE
Possible to have prompt at bottom of terminal at all times?

### ISSUE THREAD

**Issue by adambuczynski** [flatiron/prompt#166]
I have a scenario where my app connects to two different services, a local piece of hardware via serial port, and a remote server via sockets.

Once the hardware connection is established, I show a prompt where the user can enter manual commands at the terminal if needed.

This works fine, but once the server connection is established, which is usually after the device connection is already there, it logs a success message which get pasted on top of the prompt.

This is not a major issue, but kind of ugly, and results in something like this on the terminal:

```
Connecting to device on port /dev/cu.usbserial-FT8X0FP7 ...
Connecting to server at http://localhost:8081/ ...
Connected to device
Enter command: Connected to server
 //<-- prompt is now here on a new line
```

Ideally, I would like to see subsequent console log lines cover the `Enter command:` bit and for that line to move down, as follows:

```
Connecting to device on port /dev/cu.usbserial-FT8X0FP7 ...
Connecting to server at http://localhost:8081/ ...
Connected to device
Connected to server
Enter command: //<-- prompt here
```

Any thoughts on how to accomplish this?

**Comment by hawkins** [comment]
I'm running into the same issue, did you ever find a solution? I've looked at using Blessed, but it seems too complicated for my project's needs.

**Comment by adamreisnz** [comment]
Nope, never got a reply here, and didn't bother to try and solve it myself as it was a minor inconvenience only =/

**Comment by RMarjanovic** [comment]
This is quite a late reply (since I just stumbled upon your question), however I think inquirer-menu might solve your problems @adamreisnz.
